docs/framework/whats-new/index.md

```diff
@@ -1745,9 +1745,9 @@ With Ngen PDBs, NGen can create a PDB that contains the IL-to-native mapping wit
 
 - **.NET Native**
 
-  Windows apps for Windows 10 that target .NET Core and are written in C# or Visual Basic can take advantage of a new technology that compiles apps to native code rather than IL. They produce apps characterized by faster startup and execution times. For more information, see [Compiling Apps with .NET Native](../net-native/index.md). For an overview of .NET Native that examines how it differs from both JIT compilation and NGEN and what that means for your code, see [.NET Native and Compilation](../net-native/net-native-and-compilation.md).
+  Windows apps for Windows 10 that target .NET Core and are written in C# or Visual Basic can take advantage of a new technology that compiles apps to native code rather than IL. They produce apps characterized by faster startup and execution times. For more information, see [Compiling Apps with .NET Native](/windows/uwp/dotnet-native/). For an overview of .NET Native that examines how it differs from both JIT compilation and NGEN and what that means for your code, see [.NET Native and Compilation](/windows/uwp/dotnet-native/net-native-and-compilation).
```

Comment:
```suggestion
Universal Windows Platform (UWP) apps that are written in C# or Visual Basic can take advantage of a new technology that compiles apps to native code rather than IL. This technology produces apps that have faster startup and execution times. For more information, see [Compiling Apps with .NET Native](/windows/uwp/dotnet-native/). For an overview of .NET Native that examines how it differs from both JIT compilation and NGEN and what that means for your code, see [.NET Native and Compilation](/windows/uwp/dotnet-native/net-native-and-compilation).
```